modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -325,14 +326,22 @@ public void updated(Dictionary<String, ?> properties) throws ConfigurationExcept
     ingestZipJobLoad = LoadUtil.getConfiguredLoadValue(properties, ZIP_JOB_LOAD_KEY, DEFAULT_INGEST_ZIP_JOB_LOAD,
             serviceRegistry);
     // try to get overwrite series option from config, use default if not configured
+    isOverwrite = defaultOverWriteValue;
     try {
-      isOverwriteSeries = Boolean.parseBoolean(((String) properties.get(PROPKEY_OVERWRITE_SERIES)).trim());
+      String overwriteParam = (String) properties.get(PROPKEY_OVERWRITE);
+      if (overwriteParam != null) {
+        isOverwrite = BooleanUtils.toBoolean(overwriteParam.trim().toLowerCase(), "true", "false");
```

Comment:
Most services just go with the default which afaik also make the lowercase, … unnecessary:
```suggestion
        isOverwrite = BooleanUtils.toBoolean(overwriteParam);
```